### PR TITLE
Feature/core 8803 improve geo list item

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

File: src/styles/elements/GeoList/_geo-list-item.scss [hunk 1]
```diff
@@ -59,6 +64,13 @@
   align-items: center;
 }
 
+.geo-list-item__icon-and-label__label {
```
Comment (dletran) [inline src/styles/elements/GeoList/_geo-list-item.scss:67]: let's try to order the props alphabetically, I'll create a task to include the lint rules to check that in the DS

**Review by dletran — COMMENTED** [review]
what about the places where we use geo-list-item without applying these extra styles? have you checked if they also look fine now?

File: src/styles/elements/GeoList/_geo-list-item.scss (hunk 1 above)
Comment (irmakyz) [reply src/styles/elements/GeoList/_geo-list-item.scss:67]: okayy, right i forgot it, fixing

**Comment by irmakyz** [comment]
> what about the places where we use geo-list-item without applying these extra styles? have you checked if they also look fine now?

yess i checked all of them

**Review by dletran — APPROVED** [review]

**Review by fofi — APPROVED** [review]

**Pull request merged**